xds/src/main/java/io/grpc/xds/RouterFilter.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds;
+
+import com.google.protobuf.Message;
+import io.grpc.ClientInterceptor;
+import io.grpc.LoadBalancer.PickSubchannelArgs;
+import io.grpc.xds.Filter.ClientInterceptorBuilder;
+import java.util.concurrent.ScheduledExecutorService;
+import javax.annotation.Nullable;
+
+/**
+ * Router filter implementation. Currently this filter does not parse any field in the config.
+ */
+enum RouterFilter implements Filter, ClientInterceptorBuilder {
+  INSTANCE;
+
+  static final String TYPE_URL =
+      "type.googleapis.com/envoy.extensions.filters.http.router.v3.Router";
+
+  static final FilterConfig ROUTER_CONFIG = new FilterConfig() {
+    @Override
+    public String typeUrl() {
+      return RouterFilter.TYPE_URL;
+    }
+  };
+
+  @Override
+  public String[] typeUrls() {
+    return new String[] { TYPE_URL };
+  }
+
+  @Override
+  public StructOrError<? extends FilterConfig> parseFilterConfig(Message rawProtoMessage) {
+    return StructOrError.fromStruct(ROUTER_CONFIG);
```

Comment:
Why is this useful?

Reply:
We don't parse any specific field in the raw config, so we just return a dummy "parsed" config.